src/components/app/variablesV2.css

```diff
@@ -96,6 +96,7 @@ or "warn/action" ineastd of "red/green"
   --color-white-smoke: #f5f7fa;
   --color-burnt-sienna: #ec6868;
   --color-ufo-green: #2bd67b;
+  --color-amount-green: #00d563;
```

Comment:
I would rename this, since in the future it may be used to something else rather than the amount.